packages/js-sdk/package.json

```diff
@@ -11,7 +11,7 @@
     ],
     "scripts": {
         "build": "scripts/build-abi-js.sh",
-        "test": "truffle test",
+        "test": "mkdir -p reports;truffle test",
```

Comment:
do we need to add this to the ethereum-contracts tests too?

Reply:
true, but perhaps the js code itself should rather try to create the dir instead... let me write that